### DISCUSSION TITLE
Bootstrap 5 Css Confusion

### DISCUSSION THREAD

**Discussion by mxmissile** [Megabit/Blazorise#5443] [Q&A] [answered]
I'm using BS5 5.3.3 and Blazorise.Bootstrap5, given that, if I am using my own copy of bootstrap 5.3.3 do I still need to include the blazorise css?

```
<link href="/bootstrap/css/bootstrap.min.css" rel="stylesheet" />
<link href="_content/Blazorise/blazorise.css" rel="stylesheet" />
<link href="_content/Blazorise.Bootstrap5/blazorise.bootstrap5.css" rel="stylesheet" />
```

Or is this all I need?

`<link href="/bootstrap/css/bootstrap.min.css" rel="stylesheet" />`

**Comment by stsrki** [comment, marked as answer]
Yes, the `blazorise.bootstrap5.css` is needed because it contains some features and styles that are not available natively by bootstrap.
